### PR TITLE
[14.0][FIX] Fixed duplicate rules when creating a subcontractor partner

### PR DESCRIPTION
**Pull request by aayartsev**

### CODE REVIEW

**Review by elvise — APPROVED** [review]
Great!

**Comment by elvise** [comment]
@pedrobaeza take a look about this fix :)

File: mrp_subcontracting_partner_management/__manifest__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "Subcontracting Partner Management",
-    "version": "14.0.1.0.0",
+    "version": "14.0.1.1.0",
```
Comment (pedrobaeza) [inline mrp_subcontracting_partner_management/__manifest__.py]: Don't change the version manually. It's done by the bot on merge.

**Comment by pedrobaeza** [comment]
Squash commits into one, and follow the guidelines for the commit message: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#71commit-message

**Comment by aayartsev** [comment]
Dear colleagues. Please tell me how can I fix the error (codecov). It occurs to me in different situations. And I still couldn't figure out how to fix it.

**Comment by elvise** [comment]
@OCA/manufacturing-maintainers can we merge this fix ?

**Review by francesco-ooops — APPROVED** [review]
Functional review ok

**Comment by ivs-cetmix** [comment]
Will continue here https://github.com/OCA/manufacture/pull/821

**Pull request closed without merging**